example/scripts/lock-stack.js

```diff
@@ -3,48 +3,160 @@
 'use strict';
 
 /**
- * Functionality for locking stacks with expiration using the Mutex provided by
- * @cumulus/common. Stacks are locked based on the deployment name.
+ * Functionality for locking stacks with expiration and SHA tracking
  *
  * To use this, a DynamoDB table is required with the primary partition key being
  * a string - 'key' and no sort key. The table name should be set in LOCK_TABLE_NAME
  */
 
-const {
-  concurrency
-} = require('@cumulus/common');
-const aws = require('@cumulus/common/aws');
+const { aws } = require('@cumulus/common');
```

Comment:
Want to update this to use the new `aws-client` package?

Reply:
Yes, should TODO - this update was written before that was in master

Reply:
4a48680b1